internal/activity.go

```diff
@@ -145,6 +145,10 @@ type (
 		// activities directly from the workflow task back to this worker which is
 		// faster than non-eager which may be dispatched to a separate worker.
 		DisableEagerExecution bool
+
+		// VersioningIntent specifies whether this activity should run on a worker with a compatible
+		// build ID or not. See VersionIntent.
+		VersioningIntent VersionIntent
```

Comment:
use the same name for the value and type? otherwise it'll be hard to remember what it's called
```suggestion
		VersioningIntent VersioningIntent
```

Reply:
I intentionally made them different so Go's idiotic docstring format would do the right thing. Otherwise you don't get a link to the type.